README.md

```diff
@@ -17,6 +17,10 @@ git clone https://github.com/mozilla/product-details-json
 sudo yum install npm
 sudo npm install -g less 
 ```
+#### Note: The specific version of less used and known to work is 2.7.2, it can be installed via the following command:
+```
+sudo npm install -g less@2.7.2`
```

Comment:
I think you should just replace the generic install command, and change the note to something like "Less versions above 3.0 won't work." It's confusing to have to go back and change your less version if you accidentally run the generic command first on a distro that has Less 3 by default like Ubuntu 17.

Reply:
Yeah, I'll take care of that now. Were there big change in 3.0? Can we fix that so that it does work?